src/env.js

```diff
@@ -7,13 +7,11 @@
 
   // Environment name
   window.__env.ENVIRONMENT = 'local';  // local | dev | test | prod
-  window.__env.BANNER_COLOUR = 'red'
-  window.__env.API_LOCATION = 'http://localhost:3000';
-  window.__env.API_PATH = '/api';
-  window.__env.API_PUBLIC_PATH = '/api/public';
-  window.__env.KEYCLOAK_CLIENT_ID = 'eagle-admin-console';
-  window.__env.KEYCLOAK_URL = 'https://oidc.gov.bc.ca/auth';
-  window.__env.KEYCLOAK_REALM = 'eagle';
+  window.__env.API_LOCATION = 'https://pkqlwkdzka.execute-api.ca-central-1.amazonaws.com';
+  window.__env.API_PATH = '/dev';
+  window.__env.KEYCLOAK_CLIENT_ID = 'nrpti-admin';
```

Comment:
assuming this will change away from `nrpti-admin` at a later date?

Reply:
That's correct. We don't have a Parks realm yet.